src/ApiGateway/SimpleWebsocketClient.php

```diff
@@ -0,0 +1,62 @@
+<?php declare(strict_types=1);
+
+namespace Bref\ApiGateway;
+
+use AsyncAws\Core\Request;
+use AsyncAws\Core\Stream\StreamFactory;
+use Symfony\Component\HttpClient\HttpClient;
+
+/**
+ * A simple alternative for communicating with websocket connections.
+ */
+final class SimpleWebsocketClient
+{
+    /** @var string */
+    private $stage;
+
+    /** @var WebsocketClient */
+    private $client;
+
+    public function __construct(string $apiId, string $region, string $stage, int $timeout = 60)
+    {
+        $this->stage = $stage;
+        $this->client = new WebsocketClient(
+            [
+                'region' => $region,
+                'endpoint' => sprintf('https://%s.execute-api.%s.amazonaws.com', $apiId, $region),
+            ],
+            null,
+            HttpClient::create([
+                'timeout' => $timeout,
+            ])
+        );
+    }
+
+    public function disconnect(string $connectionId): bool
+    {
+        return $this->client->process(
+            $this->request('DELETE', sprintf('/%s/@connections/%s', $this->stage, $connectionId))
+        )->getStatusCode() === 200;
```

Comment:
Instead of returning booleans would it would make more sense to throw exceptions? Said another way: if the status code is `!== 200` does that mean that this is an exceptional error? Or is that a normal case?

Exceptions would seem more consistent with any other PHP class (where the default behavior works, else there's an exception).

Reply:
@mnapoli Wops - you're right. Exceptions is already thrown in case of a unsuccessful disconnect anyway - so this is really unnecessary. I'll change this up.